server/galaxyls/tests/files/simple_params_01_test.xml

```diff
@@ -0,0 +1,9 @@
+<test expect_num_outputs="$1">
+    <!--TODO: auto-generated test case. Please fill in the required values-->
+    <param name="arg_01" value="$2"/>
+    <param name="arg_02" value="$3"/>
+    <param name="param-01" value="$4"/>
```

Comment:
this name is wrong ... it needs to be `param_01`

Reply:
Ok, I see, then probably what is wrong is the name in the original input param [here](https://github.com/galaxyproject/galaxy-language-server/pull/73/commits/20a129563febbc766dd3349a7607fb7ae24f204f#diff-4a3f9be16becca0107be45b53eae3d252bcff4896f37ceaa76b84b3c9910817dR5)